dashboard.py

```diff
@@ -240,13 +240,13 @@ def get_image(n_clicks, subspecies, view, sex, hybrid):
         #print("No Such Images. Please make another selection.")
         return html.H4("No Such Images. Please make another selection.", 
                     style = {"color":"MidnightBlue"})
-    if 'D_lowres' in filename:
-        #remove 'tif' from filename and replace with 'png' in url
-        image_directory = "https://github.com/Imageomics/dashboard-prototype/blob/feature/image-options/test_data/images/dorsal_images/" + filename[:-3] + "png?raw=true"
+    if 'D_lowres' in filename:    
+        image_directory = "dorsal_images/"
     else:
-        #remove 'tif' from filename and replace with 'png' in url
-        image_directory = "https://github.com/Imageomics/dashboard-prototype/blob/feature/image-options/test_data/images/ventral_images/" + filename[:-3] + "png?raw=true"
-    return html.Img(src = image_directory)
+        image_directory = "ventral_images/"
+    #remove 'tif' from filename and replace with 'png' in url
+    image_path = "https://github.com/Imageomics/dashboard-prototype/blob/feature/image-options/test_data/images/" + image_directory + filename[:-3] + "png?raw=true"
```

Comment:
This URL is from the `feature/image-options` branch. Once this PR is merged and the branch is deleted the URL will no longer valid. I think you should change `feature/image-options` to `main`. 

Also it would be nice to declare the base URL at the top of this file and re-use that variable here. This will make the code a little easier to read and changing the URL will be simpler in the future.

Something like:
```suggestion
    image_path = IMAGES_BASE_URL + image_directory + filename[:-3] + "png?raw=true"
```

Then below the imports at the top of the file
```
IMAGES_BASE_URL = "https://github.com/Imageomics/dashboard-prototype/blob/main/test_data/images/"
```